tests/unit/document/test_converters.py

```diff
@@ -259,10 +259,6 @@ def test_convert_uri_to_data_uri(uri, mimetype):
     'uri, chunk_num',
     [
         (os.path.join(cur_dir, 'toydata/test.glb'), 1),
-        (
-            'https://storage.googleapis.com/showcase-3d-models/ShapeNetV2/airplane_aeroplane_plane_0.glb',
-            16,
-        ),
```

Comment:
why do we need to remove this ?

Reply:
if you open the file, you'll find out it was removed from google cloud storage

Reply:
I'm mostly not sure where I can find another reliable file online

Reply:
can't we just put one in a bucket somewhere? You could even download one of the images that is included in this repo, you can get a perm link to its location on github

Reply:
makes sense and we already check the .glb files with the test above so no need for this one

Reply:
thanks for the idea